templates/csharp/api-plugin-from-scratch-bearer/.{{NewProjectTypeName}}/README.md.tpl

```diff
@@ -9,14 +9,30 @@
 > - [Visual Studio 2022](https://aka.ms/vs) 17.11 or higher and [install Teams Toolkit](https://aka.ms/install-teams-toolkit-vs)
 > - A [Microsoft 365 account for development](https://docs.microsoft.com/microsoftteams/platform/toolkit/accounts)
 
-1. In the debug dropdown menu, select Dev Tunnels > Create a Tunnel (set authentication type to Public) or select an existing public dev tunnel
+### Add your own API Key
+
+1. Open PowerShell, change the current working directory to this project root and run command `./GenerateApiKey.ps1`
+    ```
+    > ./GenerateApiKey.ps1
+    ```
+
+2. The above command will output something like "Generated a new API Key: xxx...".
+3. Fill in API Key into `env/.env.*.user`.
+    ```
+    SECRET_API_KEY=<your-api-key>
+    ```
+
+### Start the app in Teams Web Client
+
+1. If you haven't added your own API Key, please follow the above steps to add your own API Key.
+2. In the debug dropdown menu, select Dev Tunnels > Create a Tunnel (set authentication type to Public) or select an existing public dev tunnel
 </br>![image](https://raw.githubusercontent.com/OfficeDev/TeamsFx/dev/docs/images/visualstudio/debug/create-devtunnel-button.png)
-2. Right-click the '{{NewProjectTypeName}}' project and select Teams Toolkit > Prepare Teams App Dependencies
-3. If prompted, sign in with a Microsoft 365 account for the Teams organization you want to install the app to.
-4. Press F5, or select the `Debug > Start Debugging` menu in Visual Studio to start your app
+3. Right-click the '{{NewProjectTypeName}}' project and select Teams Toolkit > Prepare Teams App Dependencies
+4. If prompted, sign in with a Microsoft 365 account for the Teams organization you want to install the app to.
+5. Press F5, or select the `Debug > Start Debugging` menu in Visual Studio to start your app
 </br>![image](https://raw.githubusercontent.com/OfficeDev/TeamsFx/dev/docs/images/visualstudio/debug/debug-button.png)
-5. When Teams launches in the browser, you can open the Copilot app and send a prompt to trigger your plugin.
-6. Send a message to Copilot to find an NuGet package information. For example: Find the NuGet package info on Microsoft.CSharp.
+6. When Teams launches in the browser, you can open the Copilot app and send a prompt to trigger your plugin.
+7. Send a message to Copilot to find an NuGet package information. For example: Find the NuGet package info on Microsoft.CSharp.
```

Comment:
This template is not for finding an NuGet package, maybe you should modify this part to:

```
6. When Teams launches in the browser, click the Apps icon from Teams client left rail to open Teams app store and search for Copilot.
7. Open the `Copilot` app, select `Plugins`, and from the list of plugins, turn on the toggle for your plugin. Now, you can send a prompt to trigger your plugin.
8. Send a message to Copilot to query the repair record. For example: List all repairs.
```

Reply:
Sure.